src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands_Test.py

```diff
@@ -0,0 +1,98 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Usually filename is in lowercase.

Reply:
this is used test script, i forgot to delete it before submitting the PR. I have deleted it now